downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

```diff
@@ -1,29 +1,25 @@
+// this info is duplicated here: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/installing_and_upgrading_private_automation_hub/installing_online_or_offline#doc-wrapper
 
 [id="installing-the-aap-setup-bundle_{context}"]
 
-= Installing the AAP Setup Bundle
+= Downloading and installing the {PlatformNameShort} setup bundle
 
-The “bundle” version is strongly recommended for disconnected installations as it comes with the RPM content for AAP as well as the default execution environment images that will be uploaded to your Private Automation Hub during the installation process.
+[role="_abstract"]
+The “bundle” version is strongly recommended for disconnected installations because it includes the RPM content for {PlatformNameShort} as well as the default {ExecEnvShort} images that will be uploaded to your {PrivateHubName} during the installation process.
```

Comment:
See the Red Hat Supplementary style guide in the [Word usage section](https://redhat-documentation.github.io/supplementary-style-guide/#_word_usage), specifically the word "recommend". Currently, it says "Avoid 'recommend'." Instead direct users to take the recommended action. 
```suggestion
Use the "bundle" version for disconnected installations because it includes  the RPM content for {PlatformNameShort} as well as the default {ExecEnvShort} images that will be uploaded to your {PrivateHubName} during the installation process.
```